recipes/libmagic/all/conanfile.py

```diff
@@ -0,0 +1,103 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple import fix_apple_shared_install_name
+from conan.tools.build import cross_building
+from conan.tools.env import VirtualBuildEnv, VirtualRunEnv
+from conan.tools.files import copy, get, rm, rmdir, rename
+from conan.tools.gnu import Autotools, AutotoolsToolchain
+from conan.tools.layout import basic_layout
+import os
+
+required_conan_version = ">=1.52.0"
+
+class LibmagicConan(ConanFile):
+    name = "libmagic"
+    description = "Magic number recognition library - detect files with data in particular fixed formats."
+    license = "DocumentRef-COPYING:LicenseRef-BSD-2-Clause-File"  # Modified BSD 2-Clause that states no restrictions on US export
```

Comment:
I'm inclined to think that the `DocumenRef-` portion should indicate the library or project name instead of the specific, i.e. `DocumentRef-file` instead of `DocumentRef-COPYING`. This fits better with the example in [D.3 Simple license expressions](https://spdx.github.io/spdx-spec/v2-draft/SPDX-license-expressions/#d3-simple-license-expressions). I'm not sure, but I kind of get that feeling this is supposed to be a reference to an SPDX document that we create, i.e. like the [SPDXJSONExample-v2.2.spdx.json](https://github.com/spdx/spdx-spec/blob/development/v2.2.2/examples/SPDXJSONExample-v2.2.spdx.json), but I don't think anyone has done something like that on CCI.

Reply:
@RubenRBS

Reply:
I asked it time ago: https://opensource.stackexchange.com/a/14309/31147